Dockerfile.cloudwatcher

```diff
@@ -18,11 +18,11 @@ WORKDIR /go/src/app
 # Fetch dependencies
 # Do this first for caching sake
 ENV GO111MODULE=on
-COPY go.mod go.sum ./
+COPY v2/go.mod v2/go.sum ./
 RUN go mod download
 
 # Copy source
-COPY . .
+COPY v2/ .
```

Comment:
Could we move this (and the pattern in the other files) to line 21 to keep all the `COPY` operations in a single layer?  Or would this interfere with the `go mod download`?

Reply:
The idea is that downloading all the dependencies takes a while and the source code changes more frequently (especially during local dev), so by having it separate we take advantage of the Docker build cache.